### DISCUSSION TITLE
A module not found error in a python-selenium project

### DISCUSSION THREAD

**Discussion by spoorthy24** [googleapis/google-cloud-python#12233] [Q&A] [answered]
I am actually new to this, I was assigned to do a selenium project where there's already a bunch of code written i need to run and test it out. But when i run i keep getting this error. I have given the code and the error below! can any one help me with it.
`import unittest
from common.common import minutes_seconds, cleanup_screenshots
from common.selenium_wrappers import *
import HTMLTestRunner
from Email import *
from Email import send_email
import common.global_vars
from common.common import *
import sys
import common.common`

Error ::
` from common.common import minutes_seconds, cleanup_screenshots
  File "c:\Users\admin\OneDrive\Documents\Insights\common\common.py", line 16, in <module>
    import global_vars
ModuleNotFoundError: No module named 'global_vars'`

directory:
![image](https://github.com/googleapis/google-cloud-python/assets/54950730/220a72ee-a4d7-4099-be0d-a64390a687b8)

**Comment by parthea** [comment, marked as answer]
Please use stackoverflow for general python questions: https://stackoverflow.com/questions/tagged/python
